_includes/media/_bitmovin-player.html

```diff
@@ -28,7 +28,8 @@
             image: `{{ image | imgix: site.imgix }}`,
             isCard: `{{ include.isCard }}`,
             duration: `{{ video.duration }}`,
-            isStream: `{{ include.isStream }}`
+            isStream: `{{ include.isStream }}`,
+            subtitles_url: '{{ video.transcription.url }}'
```

Comment:
Same as comment I left in crds-media

https://github.com/crdschurch/crds-media/pull/713#pullrequestreview-254763903